Misc/NEWS.d/next/Documentation/2019-10-06-23-44-15.bpo-38387.fZoq0S.rst

```diff
@@ -0,0 +1 @@
+Document PyDoc_STRVAR macro in the C-API reference.
```

Comment:
Please add in a "Patch by <your preferred name>" like this https://github.com/python/cpython/blob/master/Misc/NEWS.d/next/Core%20and%20Builtins/2019-05-28-11-47-44.bpo-37077.S1h0Fc.rst

Reply:
Thanks for reviewing and hello from Stack Overflow! 👋

Reply:
@bsolomon1124 The "Patch by <name>" is optional, but you can certainly add it if you'd like to.